src/chainer.test.js

```diff
@@ -1 +1,49 @@
 const { chainer } = require('./chainer');
+
+describe('chainer function', () => {
+  const f1 = jest.fn((x) => x * 2);
+  const f2 = jest.fn((x) => x + 2);
+  const f3 = jest.fn((x) => Math.pow(x, 2));
+
+  const mockFunctions = [f1, f2, f3];
+  const chained = chainer(mockFunctions);
+
+  it('should be declared', () => {
+    expect(chainer).toBeInstanceOf(Function);
+  });
+
+  it('should chain multiple functions correctly', () => {
+    expect(chained(0)).toBe(4);
+  });
+
+  it('should chaines functions in correct order', () => {
+    expect(chained(2)).toBe(36);
+  });
+
+  it('should calls each function in the array', () => {
+    chained(3);
+
+    expect(f1).toHaveBeenCalled();
+    expect(f2).toHaveBeenCalled();
+    expect(f3).toHaveBeenCalled();
+  });
+
+  it('should chaines functions with negative number', () => {
```

Comment:
There is a typo in the test description. It should be 'should chain functions with negative number' instead of 'should chaines functions with negative number'.